src/mmw/js/src/user/controllers.js

```diff
@@ -7,6 +7,7 @@ var App = require('../app'),
 var SignUpController = {
     signUp: function() {
         new views.SignUpModalView({
+            app: App,
```

Comment:
I did not notice when we started passing `App` as an argument. Is this to avoid circular references?